frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/EditFieldDrawer/edit-fieldtype/common/useEditFieldForm.ts

```diff
@@ -52,6 +52,8 @@ type UseEditFieldFormProps<
       output: FieldShape,
     ) => Promise<FieldShape> | FieldShape
   }
+} & {
+  mode?: 'onChange' | 'onBlur' | 'onSubmit' | 'onTouched' | 'all'
```

Comment:
suggest using the `Mode` type exported directly from 'react-hook-form` so types stay consistent when our dependencies change

Reply:
[6458379](https://github.com/opengovsg/FormSG/pull/5253/commits/645837922e0ee16de3ffba7c17884898ed70906f)